manifests/0000_90_cluster-authentication-operator_03_prometheusrule.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: authentication-operator
+  namespace: openshift-authentication-operator
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  groups:
+  - name: old-format-tokens
+    rules:
+      - alert: DeprecatedTokensInUse
+        annotations:
+          message: >-
+            Either OAuthAccessTokens or OAuthAuthorizeTokens have been detected in your cluster that do not match
+            the new required naming scheme in the next version of OpenShift. These are all such tokens
+            whose name is not prefixed by "sha256~". These tokens will not be usable in the next OpenShift
+            release and it is therefore advised to remove them.
```

Comment:
would add something like:

These tokens are deprecated since OpenShift 4.6 for security reasons (compare release notes <link>).